addons/mass_mailing/models/mailing.py

```diff
@@ -174,6 +174,7 @@ def _get_default_mail_server_id(self):
         default=_get_default_mail_server_id,
         help="Use a specific mail server in priority. Otherwise Odoo relies on the first outgoing mail server available (based on their sequencing) as it does for normal mails.")
     contact_list_ids = fields.Many2many('mailing.list', 'mail_mass_mailing_list_rel', string='Mailing Lists')
+    last_test_recipients_email = fields.Text(help='Carriage-return separated list of last attempted test emails')
```

Comment:
Would rename it `default_test_email_to` instead and add here a `default=lambda self: self.env.user.email_formatted`. It should also have a string, `Default Test Recipients` ?